web-app/js/portal/details/LayerControlPanel.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2012 IMOS
+ *
+ * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
+ *
+ */
+
+Ext.namespace('Portal.details');
+
+Portal.details.LayerControlPanel = Ext.extend(Ext.Container, {
+
+    initComponent: function() {
+        this.opacitySlider = new Portal.common.LayerOpacitySliderFixed({
+            layer: new OpenLayers.Layer("Dummy Layer"),
+            keyIncrement: 10,
+            increment: 5,
+            minValue: 20, // minimum visibility for the current layer is 20%
+            maxValue: 100,
+            aggressive: true,
+            width: 175,
+            isFormField: true,
+            inverse: false,
+            fieldLabel: OpenLayers.i18n('Opacity'),
+            plugins: new GeoExt.LayerOpacitySliderTip({
+                template: '<div class="opacitySlider" >Opacity: {opacity}%</div>'
+            })
+        });
+
+        // Put in container so that we see the label.
+        this.opacitySliderContainer = new Ext.Panel({
+            layout: 'form',
+            items: [ this.opacitySlider ]
+        });
+
+        this.layerVisibilityCheckbox = new Ext.form.Checkbox({
+            value: true,
+            boxLabel: OpenLayers.i18n('showMapLayer'),
+            checked: true,
+            listeners: {
+                scope: this,
+                check: this._visibilityButtonChecked
+            }
+        });
+
+        this.zoomToLayer = new Ext.ux.Hyperlink({
```

Comment:
I tried to keep code changes to a minimum, apart from moving things around of course.  Although I was able to get ride of a weird bit of code that didn't seem necessary anymore: https://github.com/aodn/aodn-portal/pull/1865/files#diff-8da76661a875615270e9fb526f5efac0L149